pkg/services/dns/dns_config_windows.go

```diff
@@ -3,24 +3,23 @@
 package dns
 
 import (
-	"net/netip"
+	"net"
 	"strconv"
 
 	qdmDns "github.com/qdm12/dns/v2/pkg/nameserver"
 )
 
-func getDNSHostAndPort() (string, string, error) {
+func getDNSHostAndPort() ([]string, error) {
 	nameservers := qdmDns.GetDNSServers()
 
-	var nameserver netip.AddrPort
+	var dnsServers = make([]string, 5)
```

Comment:
Why 5? You could leave this as `var dnsServers = []string{}` or `var dnsServers []string` since you are using `append` in the loop which will grow the array as needed.